doc/Caliptra.md

```diff
@@ -60,8 +60,10 @@ The Caliptra Workgroup acknowledges the following individuals for their contribu
 |January 2023|0.60|Jeff Andersen|Added additional details on hitless update attestation.|
 |January 2023|0.61|Jeff Andersen|Removed KEY MANIFEST SVN from spec.|
 |February 2023|0.62|Piotr Kwidzinski|Moved spec to Caliptra GitHub repo.|
+|May 2023|0.63|Caleb Whitehead|Updated Error Reporting and Handling HW Error Table.|
```

Comment:
Hi @pkwidzin-amd can we remove merges. Can we rebase and linearize this PR?

Reply:
done